patterns/2-structured/maturity-model.md

```diff
@@ -92,28 +75,26 @@ When working in host teams mistakes will automatically be widely visible. In ord
+* CF-0: There are no processes or established channels. Some members of the organization share materials via private channels or discussions.
+* CF-1: The organization is in the process of establishing internal guidelines and channels for encouraging diverse points of view about company/departmental decisions so that anyone belonging to the organization can use them. Some members of the organization share decision-making materials informally using unofficial platforms. Leaders maintain at least one clear and direct channel for organization members to share opinions constructively on some matters relevant to their work.
 * CF-2: The organization has established internal guidelines and channels, and provides specific resources (training programs, access to content, etc.), for encouraging and soliciting diverse points of view on team or decisions.
 * CF-3: Members of the organization share decision-making materials on officially sanctioned platforms Members of the organization share materials openly via multiple channels and methods for feedback.
-Leaders use those channels themselves, openly encourage others to use them, and maintain team-facing or public-facing records of the feedback they've received and/or the actions they've taken to address this feedback.
+Leaders use those channels themselves, openly encourage others to use them and maintain team-facing or public-facing records of the feedback they've received and/or the actions they've taken to address this feedback.
 
-**Leadership**
+#### Leadership
 
 InnerSource projects encourage employees to contribute to projects outside of the direct influence of their direct line manager. This needs a culture of trust.
 
 * LS-0: Command & control culture, within a highly hierarchical organization.
 * LS-1: Some leaders are open to receiving feedback and creating an environment where people feel safe providing it.
-* LS-2: Most leaders in the organization are open to receiving feedback and creating an environment where people feel safe providing it. Leaders show passivity about understanding whether all members feel empowered and enabled to share. Organization encourages leaders to actively seek voices not present in dialog out for inclusion.
+* LS-2: Most leaders in the organization are open to receiving feedback and creating an environment where people feel safe providing it. Leaders show passivity about understanding whether all members feel empowered and enabled to share. The organization encourages leaders to actively seek voices not present in dialog out for inclusion.
```

Comment:
```suggestion
* LS-2: Most leaders in the organization are open to receiving feedback and creating an environment where people feel safe providing it. Leaders show passivity about understanding whether all members feel empowered and enabled to share. The organization encourages leaders to actively seek out voices not present in dialog for inclusion.
```

I think the word "out" was in the wrong place here? (and old error that wasn't caught previously)